tasks/opsep/update-l2pao-to-aliased-l1pao/.env

```diff
@@ -0,0 +1,3 @@
+ETH_RPC_URL=https://optimism-sepolia.blockpi.network/v1/rpc/public	 # Fill this in with an L1 Sepolia RPC URL that has archive data access.
```

Comment:
The comment mentions the wrong chain here. Also, I don't think we need an archive node anymore (see below comment)